### DISCUSSION TITLE
Weekly Blog 1

### DISCUSSION THREAD

**Discussion by LPkkjHD** [SE-TINF22B6/Underwatch#14] [Announcements]
We are the team Underwatch. Our goal is, to create a 2,5D top-down roguelike dungeon crawler.
The roles are as follows:

|Resource|Worker|Activities|
|---|---|---|
|Martin|Product Owner|Use Case Designer|
|Benjamin|Junior Developer|Game Implementation/Level Design|
|Jeremias|Scrum Master|Website Development|
|Jakob|Senior Developer|Game Implementation|
|Ting|Database Engineer|Database Design|

Our vision is to create a little and simple, but funny game. It will be a top-down game, so we won't have to worry too much about camera movement. We want you to be able to shoot a few pixels towards enemies, that are spawning randomly on the map. 
There should be a few different maps, so it doesn't get boring.

We want to build a website that shows the Highscores of all players. For building this website, we'll use React as well as TypeSkript. 
For creating the game itsself, we planned to use libGDX and Java and the Map will be created with Tiled.

We are working agile. So we have a Sprint length of 2 Weeks and are generally using the GitHub Backlog and Kanban.

For the first Sprint we planed to setup the basic things like a initial Reactpage and learn the basics we need to start with the project.

![image](https://github.com/SE-TINF22B6/Underwatch/assets/123720714/70627a05-bbde-4054-9bb1-e4aef61215ea)

**Comment by tech-n1c** [comment]
Nice idea, this sounds very promising! You began pretty decent, as time goes on there will be lots of things to work on. We are sure that you will come up with more fantastic ideas and a great product! We like that you already distributed all tasks according to your individual skills and personal preferences. But make sure that you still distribute your knowledge accordingly to the lecture; as everyone should be able to understand the different technologies (knowledge 🏝s shouldn't come into existence).

Keep up the great work, your team2shine 💫

**Comment by Challenger3010** [comment]
Seems like you already have a good plan to work with, we like your vision of keeping things simple. Keep it up!

Your CookHub-Team.

**Comment by ghost** [comment]
What a interesting concept! I believe you'll attract a substantial and large customer base and have the ability to both sustain and continuously enhance your application. I'm also eagerly anticipating the in-game statistics; it adds an extra layer of excitement to the experience and covers all the amusement of your game I believe."
